other/materials_designer/create_grain_boundary_film.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<p>These parameters are hard to understand without more explanation:</p><ul><li>INTERFACE_GAP</li><li>XY_SUPERCELL_MATRIX</li><li>EDGE_INCLUSION_TOLERANCE</li></ul><p> </p><p>We should consider some more explanation regarding how implementation - either in code and put a link to it here, or in docs.mat3ra.com</p><p> </p><p>VISUALIZE_REPETITIONS -&gt; <code>CELL_REPETITIONS_FOR_VISUALIZATION</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>It would be great to have a visual to explain EDGE_INCLUSION_TOLERANCE and DISTANCE_TOLERANCE</p><p> </p><p>"Distance tolerance for atoms to be considering too close when merging, and previous one be removed." -&gt; "Distance tolerance for two atoms to be considered too close. Used when merging two orientations to remove the atoms of the first one.&nbsp;"</p>